signup.html

```diff
@@ -0,0 +1,63 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link rel="preconnect" href="https://fonts.googleapis.com" />
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Figtree:wght@400;600;700&display=swap"
+      rel="stylesheet"
+    />
+    <link rel="stylesheet" href="styles.css" />
+    <script src="portfolio-form-submission.js"></script>
+    <title>Sign Up</title>
+  </head>
+  <body>
+    <header>
+      <h2 class="header-options"><a href="/portfolio.html">Home</a></h2>
+    </header>
+    <main>
+      <div id="signupModal" class="signup-modal">
+        <div class="modal-content">
+          <hr class="signup-hr" />
+
+          <div class="inner-form">
+            <h2>SIGN UP</h2>
+            <form id="submit-form" action="">
+              <section>
+                <input
+                  id="name"
+                  class="input-field"
+                  type="text"
+                  placeholder="Name"
+                  value=""
+                />
+                <p id="alert-name">Please enter your name.</p>
```

Comment:
I can't seem to find the **CSS** of the alerts. Shouldn't the message be hidden by default?